Added fixedRange property for TileSparklineSkin

novisfff opened the pull request:
The **fiexdRange** property is added to ensure that the **low** and **high** values of **TileSparklineSkin** can be selected according to the user's needs to automatically change with data changes. 
Because at some point, we really don’t want it to change automatically.

HanSolo commented:
Hmm...I understand the point but would it not make sense to make use of the already existing autoScale property in this case? Meaning to say instead of introducing a new property, simply switch autoScaling off and use the minValue and maxValue fields to restrict the range.
The other idea would be to use my [TilesFX](https://github.com/HanSolo/tilesfx) library which has a more advanced version of the sparkline skin. When I've added the TileSparklineSkin and the TileKpiSkin I've created a new library that only contains tiles for dashboards. So if I need a dashboard I usually use TilesFX and add a gauge in a custom tile if I need one. I think I've added some examples of that combination in the [TilesFX demo project](https://github.com/HanSolo/tilesfxdemo)

novisfff commented:
> Hmm...I understand the point but would it not make sense to make use of the already existing autoScale property in this case? Meaning to say instead of introducing a new property, simply switch autoScaling off and use the minValue and maxValue fields to restrict the range.
> The other idea would be to use my [TilesFX](https://github.com/HanSolo/tilesfx) library which has a more advanced version of the sparkline skin. When I've added the TileSparklineSkin and the TileKpiSkin I've created a new library that only contains tiles for dashboards. So if I need a dashboard I usually use TilesFX and add a gauge in a custom tile if I need one. I think I've added some examples of that combination in the [TilesFX demo project](https://github.com/HanSolo/tilesfxdemo)

ok, thank you, I will try to use TilesFX
